src/main/kotlin/com/jetbrains/snakecharm/lang/parser/SmkStatementParsing.kt

```diff
@@ -289,13 +306,35 @@ class SmkStatementParsing(
         referenceMarker.drop()
         return false
     }
+
+    private fun isToplevelDecoratorKeyword(): Boolean {
+        return if (myBuilder.tokenText!! in TOPLEVEL_KEYWORDS) {
```

Comment:
Kotlin suggests using expression body for methods with single `return ...` statement in method body

Reply:
https://github.com/JetBrains-Research/snakecharm/pull/405/commits/4193f35c60cbd1fde137456d622bee1667584d27